nixos/modules/services/backup/restic.nix

```
@@ -292,7 +304,14 @@ in
         (name: backup:
           let
             extraOptions = concatMapStrings (arg: " -o ${arg}") backup.extraOptions;
-            resticCmd = "${backup.package}/bin/restic${extraOptions}";
+            inhibitCmd = concatStringsSep " " [
+              "${pkgs.systemd}/bin/systemd-inhibit"
+              "--mode='block'"
+              "--who='restic'"
+              "--what='sleep'"
+              "--why='Scheduled backup ${name}' "
```

```suggestion
              "--why=${escapeShellArg "Scheduled backup ${name}"} "
```

The module isn't great at escaping `name` but we should make sure that it doesn't spill over from `systemd-inhibit`s args parsing here.